recipes/containers/lighttpd.yaml

```diff
@@ -51,14 +51,27 @@ multiPackage:
     image:
         buildTools: [squashfs-tools]
         buildScript: |
+            # Remove username and group because the systemd.service already
+            # takes care of this.
+            sed -i -e '/server.username/d' -e '/server.groupname/d' \
+                etc/lighttpd/lighttpd.conf
+
             # See https://systemd.io/PORTABLE_SERVICES.html
             mkdir -p etc/systemd/system
             install -m 644 $<<lighttpd/lighttpd.service>> \
                             etc/systemd/system/lighttpd.service
+            install -m 644 $<<lighttpd/lighttpd.socket>> \
+                            etc/systemd/system/lighttpd.socket
             touch etc/machine-id
-            touch etc/os-release
-            touch etc/resolve.conf
-            mkdir -p proc sys dev run tmp var/tmp
+            ln -s ../usr/lib/os-release etc/os-release
```

Comment:
Any specific reason to put it here?

Reply:
https://www.freedesktop.org/software/systemd/man/os-release.html

Reply:
OK, I was just wondering because the portable application overview only explicitly mentions `/etc/os-release`...